compiler/rustc_data_structures/src/sync.rs

```diff
@@ -637,6 +577,8 @@ impl<T: DynSend> Deref for IsSend<T> {
 
 pub fn assert_sync<T: ?Sized + Sync>() {}
 pub fn assert_send<T: ?Sized + Send>() {}
```

Comment:
These functions should all be replaced with Dyn* variants now.

Reply:
Done.